static/src/stylesheets/amp/_d2-comments.scss

```diff
@@ -0,0 +1,50 @@
+/* Comments
+   ========================================================================== */
+
+.d2-comment-embedded {
+    background-color: #ffffff;
+    font-family: sans-serif;
+    overflow: hidden;
+    position: relative;
+    margin-bottom: 1.5em;
+    padding-top: 15px;
+    font-size: 14px;
+    padding-left: 28px; // aligns with top level comment's body
+    border-left: 8px solid $neutral-5;
+    clear: left;
+}
+
+.d2-avatar-image {
+    float: left;
+    margin-right: 15px;
+    margin-bottom: 12px;
+
+    @include mq(tablet) {
+        width: 60px;
+        height: 60px;
+    }
+}
+
+.d2-right-col {
+    position: relative;
+    top: -3px;
+    padding-right: 16px; // prevent colliding with badge
+}
+
+.d2-username {
+    font-family: $f-serif-text;
```

Comment:
We can't use our normal fonts for AMP. Best to stick with a fallback. If a font is really important I would chat with @zeftilldeath